app/src/components/OutputsTable/VariantEditor.tsx

```diff
@@ -132,6 +132,7 @@ export default function VariantEditor(props: { variant: PromptVariant }) {
         wordWrapBreakBeforeCharacters: "",
         quickSuggestions: true,
         readOnly: !canModify,
+        scrollBeyondLastLine: false,
```

Comment:
I'm worried that if there's a long last line it could get hidden behind the save button with no way to see it? Also I like the default behavior that lets you scroll past the code, is there a strong reason to remove it?

Reply:
I keep accidentally scrolling the code editor when I intend to scroll the screen, which ends with me staring at a few syntax characters, like this:

<img width="651" alt="Screenshot 2023-09-04 at 5 06 47 PM" src="https://github.com/OpenPipe/OpenPipe/assets/41524992/bc33c5db-705d-45d6-b5aa-cacdf3a4d8f2">

However, I don't feel that strongly, so I'll revert this change in the VariantEditor. I do want to disable scrolling beyond the last line for the JSON editor in the dataset entry drawer.